lang/en/mutation.php

```diff
@@ -94,7 +94,9 @@
     'thaw.args.tokenId' => 'The token ID to thaw.',
     'thaw.description' => 'Thaw a previously frozen collection or token.',
     'transfer_all_balance.description' => 'Transfers all balances of an account to another. You can pass a keepAlive argument if you want to keep at least the existential deposit.',
-    'transfer_balance.description' => 'Transfers a balance from one account to another. You can pass the keepAlive argument if you want to check if the account will be left with at least the existential deposit.',
+    'transfer_balance.description' => '(DEPRECATED) Transfers a balance from one account to another.',
```

Comment:
Please could you also add the `deprecationReason` field to the mutation attributes as well with a note explaining why it's deprecated and what to use instead 🙂

Reply:
Tested here and it works for the whole mutation indeed. I will make the change.